HIMIKO/.htaccess

```diff
@@ -1,4 +1,4 @@
 Options +FollowSymLinks
 RewriteEngine on
 
-RewriteRule ^(.*)$ https://junjun7613.github.io/MicroKnowledge/$1 [R=302, L]
\ No newline at end of file
+RewriteRule ^himiko.owl$ https://junjun7613.github.io/MicroKnowledge/himiko.owl [R=302, L]
```

Comment:
I do not think you want to add the `hiimiko.owl` in the first part of the rewrite rule. Otherwise your w3id will be:
`https://w3id.org/HIMIKO/himiko.owl` only

Reply:
I re-uploaded the original but removing the space in the options.

Reply:
Hope this will work well..